docs/changelog/4.1.rst

```diff
@@ -671,7 +671,7 @@ Changelogs for 4.1.x
     :tags: Internals, Bug Fixes
     :pullreq: 5917
 
-    Use ``_exit()`` when we really really want to exit, for example
+    Use ``_exit()`` when we really want to exit, for example
```

Comment:
It's possible that `really really` is an intentional stress (and that I wasn't awake enough when I made these commits):
```suggestion
    Use ``_exit()`` when we really really want to exit, for example
```

Reply:
We'll go w/ some stress instead:
```suggestion
    Use ``_exit()`` when we *really* want to exit, for example
```